tpp-public/style.css

```diff
@@ -0,0 +1,135 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+    margin: 0;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    background-color: #E4EAFF;
+
+}
+
+.container {
+    grid-template-columns: repeat(6, 1fr);
+    display: grid;
+}
+
+
+header {
+    grid-column: 1/7;
+    background-color: #E4EAFF;
+    display: flex;
+    justify-content: center;
+
+}
+
+
+.profileImage {
+    grid-column: 2/3;
+}
+
+.talentInfo {
+    grid-column: 3/4;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+
+
+.email {
+    display: flex;
+    grid-column: 4/6;
+    justify-content: center;
+    align-items: center;
+}
+
+
+.phototitle {
+    display: flex;
+    grid-column: 2/7;
+    justify-content: flex-start;
+    margin-top: 60px;
+
+}
+
+.pic {
+    border-left: 10px solid;
+    border-right: 10px solid;
+    border-top: 20px solid;
+    border-bottom: 60px solid;
+    border-color: whitesmoke;
+    margin-bottom: 60px;
+    object-fit: cover;
+
+
+}
+
+.addimg {
+    border-left: 10px solid;
+    border-right: 10px solid;
+    border-top: 20px solid;
+    border-bottom: 60px solid;
+    border-color: whitesmoke;
+    margin-bottom: 60px;
+    object-fit: cover;
+    cursor: pointer;
+}
+
+.addimg:hover {
+    opacity: 0.5;
+}
+
```

Comment:
don't just call label this will change all labes as we will have 1 style for everything